eng/pipelines/templates/stages/archetype-cpp-release.yml

```diff
@@ -1,3 +1,5 @@
+#cspell: words gearama
```

Comment:
Why isn't gearama in the global cspell dictionary, along with the other team members?

Reply:
Alternative version of the above: Why are ahsonkan, Osterman, and antkmsft in the global cspell dictionary?

Reply:
Thank you! Makes sense. Done.